ts/src/renderer/phaser/scenes/GameScene.ts

```diff
@@ -13,6 +13,10 @@ class GameScene extends Phaser.Scene {
 		canvas.style.backgroundColor = 'transparent';
 		//canvas.style.pointerEvents = 'none'; // TODO remove after pixi is gone
 
+		if (ige.isMobile) {
+			this.scene.launch('MobileControls');
```

Comment:
perhaps this could be a good entry point for `ige.client(addComponent(MobileControls))`?

Reply:
also would not be, taro components should be instantiated in taro code, not in rendering code.